readme.md

```diff
@@ -1 +1,31 @@
+# NewApp Application
+
+The functionalities supported are:
```

Comment:
Functionalities and the APIs tied to it can be coupled together. Also, more details about request-response structure of APIs would help.